packages/core/src/components/HierarchyPicker.story.tsx

```diff
@@ -0,0 +1,304 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import HierarchyPicker, { Props } from './HierarchyPicker';
+
+const demoItems = [
+  {
+    name: 'Account',
+    label: 'Compte',
+    readonly: true,
+    items: [
+      {
+        name: 'Access',
+        readonly: true,
+        items: [
+          {
+            name: 'General',
+          },
+          {
+            name: 'Suspension',
```

Comment:
Can we update all the copy for these items to be a bit more... nice? And in English. Can even just be something like "Item #1".